### ISSUE TITLE
sshkey accepts any input and deletes any key that matches input 

### ISSUE THREAD

**Issue by irisb1701** [treehouses/cli#467]
```
root@treehouses:~/cli# ./cli.sh sshkey add "hello"
====== Added to 'pi' and 'root' user's authorized_keys ======
hello
root@treehouses:~/cli# ./cli.sh sshkey list
==== root keys ====
hello
==== pi keys ====
hello
root@treehouses:~/cli# ./cli.sh sshkey delete "h"
h is deleted.
root@treehouses:~/cli# ./cli.sh sshkey list
==== root keys ====
==== pi keys ====
root@treehouses:~/cli# 


```

**Comment by kjong** [comment]
It seems like this issue was fixed with PR #490?

**Comment by kjong** [comment]
Is there any point in keeping the `sshkey delete` function when `sshkey github deleteuser` was added?

If yes, I'll open a PR with the changes to make `sshkey delete` correctly match the input.
